docs/cwl/running.rst

```diff
@@ -80,9 +80,9 @@ reference ``$(runtime.tmpdir)`` in CWL tools and workflows.
 written. References to these and other output types will be in the JSON object
 printed to the stdout stream after workflow execution.
 
-``--logFile``: Path to the main logfile with logs from all jobs.
+``--logFile``: Path to the main logfile.
 
-``--writeLogs``: Directory where all job logs will be stored.
+``--writeLogs``: Directory where job logs will be stored. At ``DEBUG`` log level, this will contain logs for each Toil job run, as well as ``stdout``/``stderr`` logs for each CWL job, for output the workflow didn't capture.
```

Comment:
A CWL `Workflow` description can't capture stdout/stderr. Better to say something like "... for output not redirected to a file by use of `stderr` and/or `stdout` directives in CWL `CommandLineTool` descriptions."

https://www.commonwl.org/user_guide/topics/outputs.html#capturing-standard-output